doc/userguide/rules/vlan-keywords.rst

```diff
@@ -0,0 +1,65 @@
+VLAN Keywords
+=============
+
+.. role:: example-rule-action
+.. role:: example-rule-header
+.. role:: example-rule-options
+.. role:: example-rule-emphasis
+
+vlan.id
+-------
+
+Suricata has a ``vlan.id`` keyword that can be used in signatures to identify
+and filter network packets based on Virtual Local Area Network IDs. By default,
+it matches all layers if a packet contains multiple VLAN layers. However, if a
+specific layer is defined, it will only match that layer.
+
+VLAN id values must be between 1 and 4094. The maximum number of layers
+supported per packet is 3, and the vlan.id keyword supports negative index
+values to access layers from back to front.
```

Comment:
After Shivani's comment and thinking about how to make these limits easier to spot, how about we have something like a table that indicates accepted ranges for vlan id values and for the layers? Then it's more visible then in a paragraph 🤔

Reply:
Thank you for the suggestion, Juliana! Could you please take a look at this and let me know if I'm on the right track?
I also included the "count" value that Philippe suggested I implement. It matches the number of VLAN layers, for example, ``vlan.id:3,count`` will match if the packet has 3 VLAN layers. I'm still a bit unsure about how to document it. 

## VLAN layer

| **Values**       | **Description**       |
|----------------|---------------------|
| [default]    | match all layers |
| 0 - 2            | match specific layer |
| -3 - -1          | match specific layer with back to front indexing |
| all                | match only if all layers match |
| count          | match if the number of layers match|

## VLAN id

| **Values**       | **Description**       |
|----------------|---------------------|
| 1 - 4094   | valid range for vlan id |
| 1 - 3            | valid range of number of layers |

Reply:
I think these do help in finding and visualizing the important info ✨ 
Trying to think a bit the best way to represent the numeric ranges... maybe "`-3` -- `-1`", for instance, to highlight what are the values?

To help with reading and interpreting, I suggest having first the VLAN id table, as that's the first value, and then the layer one.

For the `count` case -- maybe "match on the number of layers"? and "valid range of number of layers (with `count`"

Reply:
`valid range of number of layers` should be `0-3` ;-)